### PR TITLE
Fix for -t option in clevis luks bind

### PR DESCRIPTION
**Pull request by jbaublitz**
@sergio-correia Here is the fix. I've tested it locally and added a test as we discussed.

### CODE REVIEW

**Comment by jbaublitz** [comment]
I'm having a little trouble with the testing infrastructure but hopefully this helps!

File: src/luks/tests/bind-luks2 [hunk 1]
```diff
@@ -47,3 +47,7 @@ new_device "luks2" "${DEV}"
 if ! clevis luks bind -d "${DEV}" -t "$TOKEN_ID" tang "${CFG}" <<< "${DEFAULT_PASS}"; then
     error "${TEST}: Binding is expected to succeed when given a correct (${DEFAULT_PASS}) password." >&2
 fi
+
+if ! cryptsetup token export --token-id=5; then
```
Comment (sergio-correia) [inline src/luks/tests/bind-luks2:51]: The issue is here: https://github.com/latchset/clevis/pull/297/checks?check_run_id=2116394266#step:7:5079
You are missing the device in the `cryptsetup` command.

File: src/luks/tests/meson.build [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # We use jq for comparing the pin config in the clevis luks list tests.
 jq = find_program('jq', required: false)
+# We use cryptsetup for testing LUKS2 binding and saving the token in a given token slot.
+cryptsetup = find_program('cryptsetup', '/usr/sbin/cryptsetup', required: true)
```
Comment (sergio-correia) [inline src/luks/tests/meson.build:4]: This should not be required.

File: src/luks/tests/bind-luks2 (hunk 1 above)
Comment (jbaublitz) [reply src/luks/tests/bind-luks2]: Whoops! Sorry about that. This should be resolved.

**Comment by jbaublitz** [comment]
The most recent test failures were what I first saw which is why I made the edits. They don't appear to be related to my changes as there are errors like `cryptsetup not found`.

**Comment by sergio-correia** [comment]
> The most recent test failures were what I first saw which is why I made the edits. They don't appear to be related to my changes as there are errors like `cryptsetup not found`.

Yeah, I see it now. It looks like `cryptsetup` is not installed in rawhide anymore -- we were lucky before that it was installed as a dependency of something else --, so let's be explicit and add it there: https://github.com/latchset/clevis/blob/master/.github/workflows/install-dependencies#L36
I will update the spec file later to add it as a build dependency.

**Comment by sergio-correia** [comment]
@jbaublitz: please, rebase, as this build issue should have been fixed by #298

**Comment by jbaublitz** [comment]
@sergio-correia Okay, rebased!

**Review by sergio-correia — APPROVED** [review]

**Comment by sergio-correia** [comment]
Thanks!

**Pull request merged**